<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: Copyright (c) Kitware Inc. -->
<!-- SPDX-License-Identifier: BSD-3-Clause -->
<ui version="4.0">
 <class>SESAMEConversionsPanelWidget</class>
 <widget class="QWidget" name="SESAMEConversionsPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="3" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>384</width>
        <height>107</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="pqExpandableTableView" name="PropertiesView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="SESAME">
       <property name="toolTip">
        <string>Set factors based on SESAME default</string>
       </property>
       <property name="text">
        <string>SESAME</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SI">
       <property name="toolTip">
        <string>Set factors based on SI</string>
       </property>
       <property name="text">
        <string>SI</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CGS">
       <property name="toolTip">
        <string>Set factors based on CGS</string>
       </property>
       <property name="text">
        <string>CGS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Custom">
       <property name="toolTip">
        <string>Set custom factors</string>
       </property>
       <property name="text">
        <string>Custom</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="ConversionFileLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Filename of Conversions File</string>
       </property>
       <property name="text">
        <string>File Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ConversionFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Filename of Conversions File</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="RestoreDefaultConversionsFile">
       <property name="toolTip">
        <string>Restore default conversion file</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Qt/Components/Resources/pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqReset.svg</normaloff>:/pqWidgets/Icons/pqReset.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ConversionFileButton">
       <property name="toolTip">
        <string>Load Conversions file</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../Qt/Components/Resources/pqComponents.qrc">
         <normaloff>:/pqWidgets/Icons/pqOpen.svg</normaloff>:/pqWidgets/Icons/pqOpen.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqExpandableTableView</class>
   <extends>QTableView</extends>
   <header>pqExpandableTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Qt/Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
